core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -653,6 +631,14 @@ public void operationComplete(ChannelFuture f) throws Exception {
                     primaryActivePort = actualPort;
                 }
 
+                if (localAddress.getAddress().isAnyLocalAddress() ||
+                    localAddress.getAddress().isLoopbackAddress()) {
+                    logger.info("Serving {} at {} - {}://127.0.0.1:{}/",
```

Comment:
This is a good change :) FWIW, it might be considered a breaking change as it will result in double-logging for production users of armeria who almost certainly are doing the logging themselves.

Reply:
I'm curious if we should log at DEBUG level.

Reply:
Let me just leave this as it is for the sake of beginners' convenience. :-)

Reply:
Yeah I think it's good to use `INFO` but should probably mention in the release notes.